app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+
+  skip_before_action :set_project, if: -> { params[:project_id].blank? }
+  skip_before_action :set_nodes, if: -> { params[:project_id].blank? }
+
+  layout :set_layout
+  def index
+    @default_columns = ['Title', 'Methodology', 'Due Date', 'Assigned']
+
+    if params[:project_id].present?
+      cards = current_project.methodology_library.boards.map(&:cards)[0]
+      @local_storage_key = "project.ce.project_#{current_project.id}.tasks_datatable"
```

Comment:
is the `ce` needed here?

Reply:
What about `"project#{params[:project_id]}.tasks_datatable"`
This would work regardless of whether `params[:project_id]` is present or not?

Reply:
This is consistent with the other local storage keys we define. In CE we need `project.ce.project_#.tasks_databale` and in pro, we need `pro.tasks_datatable`

Reply:
I removed the pro bit since that should only be in pro. It's moved to the pro-changes branch.